plugins/no-content/no-content.js

```diff
@@ -0,0 +1,36 @@
+const stylelint = require('stylelint');
+
+const ruleName = 'plugin/no-content';
+
+const messages = stylelint.utils.ruleMessages(ruleName, {
+  rejected: 'You must not use the `content` property to hard-code unlocalized text',
+});
+
+module.exports = stylelint.createPlugin(ruleName, (primaryOption) => {
+  return function(root, result) {
+    const validOptions = stylelint.utils.validateOptions(result, ruleName, {
+      actual: primaryOption,
+    });
+
+    if (!validOptions) {
+      return;
+    }
+
+    const allowedContentValues = [
```

Comment:
Can this be an overridable setting?
And can those values be strings OR regexes?

e.g. to allow `url(.*)` if for some reason it's needed.

Reply:
Yeah that sounds like a good idea. Will do.